code/index.html

```diff
@@ -2,29 +2,101 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Ski Tripz</title>
+    <link rel=stylesheet href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Sedgwick+Ave&display=swap" rel="stylesheet">  
+    <script src="https://kit.fontawesome.com/9c557cf264.js" crossorigin="anonymous"></script>
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header class="image-header">
+      <div><i class="fab fa-instagram"></i>
+        <i class="fab fa-facebook-square"></i>
+        <i class="fab fa-twitter-square"></i></div>
+      <div class="header-title"><h1><a href="https://www.youtube.com/watch?v=fvk4PpFORD4" target="_blank">Ski Tripz</a></h1></div>
+      <div class="inspo"><p>Inspo up here!<span style='font-size:12px;'>&#10548;</span></p></div>
+      <div class="header-text"><h2>WIN an awesome skiing adventure!</h2></div>
+    </header>
+ 
+    <!-- The class 'parent' is the parent to classes 'form-container' and 'accordion'-->
+    <section class="parent">
+      <div class="form-container">
+        <h2>Sign up to participate</h2><br>
+          <form action="https://httpbin.org/anything" method="post" target="_blank">
+            <label for="firstname">First Name:</label><br>
+            <input type="text" id="firstname" name="firstname" placeholder="Enter your first name" required><br>
+            <label for="lastname">Last Name:</label><br>
+            <input type="text" id="lastname" name="lastname" placeholder="Enter your last name" required><br>
 
-    <!-- Signup form -->
+            <label for="email">Email:</label><br>
+            <input type="email" id="email" name="email" placeholder="Enter your e-mail" required ><br><br>
```

Comment:
Next time, try to avoid the `<br>`-tags. I used them too but I've heard it's more 'clean code' to add some css to it

Reply:
Thank you for noticing! I actually made a note for myself to remove the <br> tags but then I totally forgot about it... I will remember it from now on ;)